packages/manager/modules/bm-server-components/src/network-interfaces/attach/interfaces-attach.module.js

```diff
@@ -1,11 +1,9 @@
 import component from './interfaces-attach.component';
-import routing from './interfaces-attach.routing';
 
 const moduleName = 'ovhManagerDedicatedServerInterfacesAttach';
 
 angular
-  .module(moduleName, ['ui.router'])
-  .config(routing)
+  .module(moduleName, ['ovh-api-services'])
```

Comment:
ovh-api-services really needed ? Maybe you have the possibility to change the calls by $http and api endpoint ?

Reply:
yes, the controller uses  `OvhApiVrack`